internal/autowire/parse.go

```diff
@@ -57,6 +58,8 @@ func (p *providerSetSrc) description(fset *token.FileSet, typ types.Type) string
 			kind = "struct provider"
 		}
 		return fmt.Sprintf("%s %s(%s)", kind, quoted(p.Provider.Name), fset.Position(p.Provider.Pos))
+	case p.AutoBinding != nil:
+		return fmt.Sprintf("wire.AutoBind (%s)", fset.Position(p.AutoBinding.Pos))
```

Comment:
Should this be `autowire.AutoBind` in the print statement?

Reply:
I believe you are correct!

Reply:
Fixed!